tests/helpers.py

```diff
@@ -314,43 +243,19 @@ def set_framework_variation(self, slug):
         db.session.commit()
 
 
-class JSONTestMixin(object):
-    """
-    Tests to verify that endpoints that accept JSON.
-    """
-    endpoint = None
-    method = None
-    client = None
-
-    def open(self, **kwargs):
-        return self.client.open(
-            self.endpoint.format(self=self),
-            method=self.method,
-            **kwargs
-        )
-
-    def test_non_json_causes_failure(self):
-        response = self.open(
-            data='this is not JSON',
-            content_type='application/json')
-
-        assert_equal(response.status_code, 400)
-        assert_in(b'Invalid JSON',
-                  response.get_data())
-
-    def test_invalid_content_type_causes_failure(self):
-        response = self.open(
-            data='{"services": {"foo": "bar"}}')
+class HelpersMixin(object):
+    def string_to_time_to_string(self, value, from_format, to_format):
```

Comment:
These methods aren't really using `self` so maybe we should make them functions?